about.html

```diff
@@ -0,0 +1,149 @@
+
+        <!--logo section -->
+        <section class="logo-exposition">
+            <div class="container">
+                <div class="logo-exposition-content">
+                    <h3 class="logo-exposition-title">Creative Commons Global Summit 2020 </h3>
+                    <p class="logo-exposition-p">
+                        A fun festival where people from more than 80 countries
+                        who believe in the value of sharing and openness and create
+                        positive change gather and share ideas will be held in Montreal in September.
+
+                    </p>
+                    <h1 class="logo-exposition-l logo">event<span>Work</span></h1>
+                </div>
+            </div>
+        </section>
+
+        <!-- history section -->
+        <section class="history">
+            <div class="container">
+                <div class="history-content">
+                    <div class="history-content-desc">
+                        <h3 class="section-title history-title ">View past global summits</h3>
+                        <p class="history-p">
+                            Take a look at the recent global summits held in Buenos Aires, Argentina
+                            and Warsaw, Poland.
+                    </div>
+                    <div class="history-cards">
+                        <div class="history-card card-1">
+                            <div class="card-overlay">
+                                <div class="history-card-content">
+                                    <h3 class="history-card-title">2020</h3>
+                                    <p class="history-card-p">
+                                        Global Summit 2020 in Buenos.
+                                    </p>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="history-card card-2">
+                            <div class="card-overlay">
+                                <div class="history-card-content">
+                                    <h3 class="history-card-title">2018</h3>
+                                    <p class="history-card-p">
+                                        Global Summit 2018 in Warsaw.
+                                    </p>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+    </div>
+    <footer class="footer about-footer">
+        <div class="container">
+            <div class="footer-content">
+                <div class="footer-logo">
+                    <h1 class="logo">Global<span>Summit</span></h1>
+                </div>
+                <div class="footer-adress">
+                    <p>2022 Creative Commons Montreal <br> Some Rights Reserved</p>
+                </div>
+            </div>
+        </div>
+    </footer>
```

Comment:
- [ ] Kindly update the footer with the right background-color and the positioning of the text as the previous reviewer asked 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/190827172-1f4a1aa6-ccab-48f0-88b4-dd212be57f39.png)

## Guidelines:
![image](https://user-images.githubusercontent.com/50721479/190827184-944eebe9-e502-4147-bf9f-300c30d0bf2d.png)